iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -1020,11 +1020,21 @@ public override async Task<ClientPropertiesUpdateResponse> SendPropertyPatchAsyn
             request.MqttTopicName = TwinPatchTopic.FormatInvariant(rid);
 
             using Message message = await SendTwinRequestAsync(request, rid, cancellationToken).ConfigureAwait(false);
-            return new ClientPropertiesUpdateResponse
+
+            bool requestIdPresent = message.Properties.TryGetValue(RequestIdKey, out string requestIdRetrieved);
+            bool versionPresent = message.Properties.TryGetValue(VersionKey, out string versionRetrievedAsString);
+
+            var response = new ClientPropertiesUpdateResponse();
+            if (requestIdPresent)
```

Comment:
You could inline all the calls to TryGetValue, rather than creating boolean return values you later reference.